examples/server/index.js

```diff
@@ -0,0 +1,56 @@
+import express from 'express';
+import AWS from 'aws-sdk';
+import bodyParser from 'body-parser';
+
+const app = express();
+
+app.use((req, res, next) => {
+  res.set('Access-Control-Allow-Methods', 'GET,POST');
+  res.set('Access-Control-Allow-Headers', 'Content-Type,X-Amz-Date,X-Api-Key,X-Amz-Security-Token,Authorization,identityId,requestId');
+  res.set('Access-Control-Allow-Origin', '*');
+
+  next();
+});
+
+app.use((req, res, next) => {
+  res.type('application/json; charset=utf-8');
+  next();
+});
+
+// JSON body parsing
+app.use(bodyParser.json());
+
+const cognito = new AWS.CognitoIdentityServiceProvider({ apiVersion: '2016-04-19', region: 'us-east-1' });
```

Comment:
Not quite sure about apiVersion here. I think it couldn't be added in .env file.